### PR TITLE
From treasury to vault

### PR DESCRIPTION
**Pull request by cboydstun**
Renamed keywords, replaced file, and more clearly defined the Treasury App. Happy Monday! Thanks for your help! =)

### CODE REVIEW

File: main/getting-started/beta.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ Don't worry, this'll get much easier soon. After it passes reviews, the wallet w
 be available via a downloadable application. On mainnet launch, Agoric will be integrated 
 with leading wallet providers via a simple plugin.
 
-## Head to the Treasury ##
+## Head to the Treasury App ##
```
Comment (Chris-Hibbert) [inline main/getting-started/beta.md:24]:
```suggestion
## Head to the Vault App ##
```

File: main/getting-started/beta.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Don't worry, this'll get much easier soon. After it passes reviews, the wallet w
...
 
 Play around with the mock assets we've kindly pre-stocked your wallet with.
```
Comment (Chris-Hibbert) [inline main/getting-started/beta.md:26]: Line 28: 

`[Treasury Application](https://treasury.agoric.app)`

should be

`[Vault Application](https://treasury.agoric.app)`

(The link only works as treasury.agoric.app currently.)

File: main/zoe/guide/contracts/vault.md [hunk 3]
```diff
@@ -95,7 +95,7 @@ collateral types and specify the parameters for its loans.
 
 ### Vats
 
-Currently the treasury runs all the `vaults` in a single vat. We intend to split
+Currently the Vault runs all the `vaults` in a single vat. We intend to split
```
Comment (Chris-Hibbert) [inline main/zoe/guide/contracts/vault.md:98]:
```suggestion
Currently the VaultFactory runs all the `vaults` in a single vat. We intend to split
```

**Review by Chris-Hibbert — CHANGES_REQUESTED** [review]

File: main/getting-started/beta.md (hunk 2 above)
Comment (cboydstun) [reply main/getting-started/beta.md:26]: okay no worries I figured we could just continue to call it the "Treasury App" until we got the URL changed. I'll fix it! 😊

**Review by Chris-Hibbert — APPROVED** [review]

**Pull request merged**